internal/testutil/env.go

```diff
@@ -61,3 +64,9 @@ func Chdir(t TestingT, dir string) string {
 
 	return wd
 }
+
+// Helper to get absolute path to testdata file.
+// It only able to helps if case Chdir() above was called or directory was not changed at all.
+func TestData(filename string) string {
+	return filepath.Join(os.Getenv("TESTS_ORIG_WD"), filename)
```

Comment:
This relies on the value being empty if `t.Chdir` is not used.

Can you make that explicit in a comment or condition check?

Reply:
added a comment